.devcontainer/Dockerfile

```diff
@@ -58,3 +58,4 @@ RUN curl -sSL https://install.python-poetry.org | python3 -
 
 # Install nox and pre-commit to automate CI stuff
 RUN pip install --user --upgrade nox nox_poetry pre-commit
+RUN git config --global --add safe.directory /workspaces/cf2tf
```

Comment:
are you on windows?

Reply:
just curious, I think I have seen my co-workers do this same thing on my devcontainers at work.

Reply:
I am on MacOS Sonoma 14.6.1. 

The issue is that in devcontainers setup,

```
"postCreateCommand": "poetry install && pre-commit install",
```

pre-commit install fails due to permission issues. If I had to guess, the git files being owned by my local user causes a permissions conflict with devcontainer user.